### PR TITLE
docs: add standalone details to NG0302

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
improve the error information of NG0302 by including details about
standalone pipes (and components)

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Issue

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
 - @AndrewKushnir :slightly_smiling_face::+1:

### CODE REVIEW

File: aio/tools/transforms/templates/error/error.template.html [hunk 1]
```diff
@@ -17,6 +17,7 @@ <h1>{$ doc.code $}: {$ doc.shortDescription $}</h1>
     allowfullscreen></iframe>
   </div>
   {% endif%}
+  {% if doc.videoInformation %}{$ doc.videoInformation | marked $}{% endif%}
```
Comment (dario-piotrowicz) [inline aio/tools/transforms/templates/error/error.template.html:20]: @gkalpak I needed to do this in order to add some text below the video :sweat: 

Screenshot at 2022-05-21 23-07-52

What do you think? is this ok?

File: aio/content/errors/NG0302.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 @shortDescription Pipe not found!
 
 @description
 Angular can't find a pipe with this name.
+
 The pipe referenced in the template has not been named or declared properly.
-A [pipe](guide/pipes) must be either declared or imported in the `NgModule` where it is used, and the name used in a template must match the name defined in the pipe decorator.
+
+In order for a [pipe](guide/pipes) to be used:
+ - it must be declared (as part of an `NgModule` or as standalone).
```
Comment (AndrewKushnir) [inline aio/content/errors/NG0302.md:13]:
```suggestion
 - it must be declared as a part of an `NgModule` (added to the `declarations` array) or marked as standalone (by adding the `standalone: true` flag to the Pipe decorator).
```

File: aio/content/errors/NG0302.md [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
+ - it must be imported in the `NgModule` or standalone component where it is used.
```
Comment (AndrewKushnir) [inline aio/content/errors/NG0302.md:14]:
```suggestion
 - it must be imported in an `NgModule` or a standalone component where it is used.
```

**Review by AndrewKushnir — COMMENTED** [review]
@dario-piotrowicz looks great, thanks for updating the guide 👍 I've left a few comments below. Thank you.

**Review by AndrewKushnir — APPROVED** [review]
👍

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit ba4f3aabeeb24f5304fbe5f5f1fc74b337f28316.

**Pull request closed without merging**